### ISSUE TITLE
k8s-vxlan tunnel profile definition missing 

### ISSUE THREAD

**Issue by ajsq** [F5Networks/k8s-bigip-ctlr#1662]
#### Description  
  
On the "CIS Installation" page, under "Creating VXLAN Tunnels on Kubernetes Cluster": 

https://clouddocs.f5.com/containers/latest/userguide/cis-helm.html#creating-vxlan-tunnels-on-kubernetes-cluster

the tunnel definition in step 2 references a profile named `k8s-vxlan` but there is no definition for that profile.

This is also the case on the "Kubernetes" page:

https://clouddocs.f5.com/containers/latest/userguide/kubernetes/#creating-vxlan-tunnels-on-kubernetes-cluster

**Comment by mdditt2000** [comment]
@ajsq i will get these examples looked at. Here is a example of how to create the vxlan configuration on the BIG-IP using tmsh

1. create a VXLAN tunnel profile. In my example I used k8s. This will be a tenant that CIS will use to add the configuration
tmsh create auth partition k8s

2. Create a VXLAN tunnel. In the example I used fl-vxlan as the tunnel name/profile. 

tmsh create net tunnels vxlan **fl-vxlan** port 8472 flooding-type none
tmsh create net tunnels tunnel **fl-vxlan** key 1 profile **fl-vxlan** local-address 192.168.200.91

![image](https://user-images.githubusercontent.com/12733938/105783189-c008cf00-5f2a-11eb-9db4-f74024b1f44f.png)

3. Create the VXLAN tunnel self-IP

tmsh create net self 10.244.20.91 address 10.244.20.91/255.255.0.0 allow-service none vlan fl-vxlan

![image](https://user-images.githubusercontent.com/12733938/105783497-753b8700-5f2b-11eb-9c5e-7e7ad83e347a.png)

![image](https://user-images.githubusercontent.com/12733938/105783564-98fecd00-5f2b-11eb-8646-906d9f266b52.png)

Let me know if you have additional questions.
